romeo_dcm/romeo_dcm_control/launch/romeo_dcm_position_control.launch

```diff
@@ -12,8 +12,10 @@
      /romeo_dcm/LHipYawPitch_position_controller /romeo_dcm/LHipRoll_position_controller /romeo_dcm/LHipPitch_position_controller /romeo_dcm/LKneePitch_position_controller /romeo_dcm/LAnklePitch_position_controller /romeo_dcm/LAnkleRoll_position_controller
      /romeo_dcm/RHipYawPitch_position_controller /romeo_dcm/RHipRoll_position_controller /romeo_dcm/RHipPitch_position_controller /romeo_dcm/RKneePitch_position_controller /romeo_dcm/RAnklePitch_position_controller /romeo_dcm/RAnkleRoll_position_controller
      /romeo_dcm/joint_state_controller
-     /romeo_dcm/NeckYaw_position_controller /romeo_dcm/NeckPitch_position_controller /romeo_dcm/HeadPitch_position_controller /romeo_dcm/HeadRoll_position_controller
-     /romeo_dcm/TrunkYaw_position_controller
      "/>
      
+    <!-- 
```

Comment:
don't move lines around: you can comment inside <node
It would make things easier to track.